### ISSUE TITLE
Openstack: Migrate existing clusters to out-of-tree CCM and CSI

### ISSUE THREAD

**Issue by alvaroaleman** [kubermatic/kubermatic#4758]
**User Story**

The internal cloud providers will get removed, hence we have to use the external ones. This means that we have to migrate existing clusters.

There is some minimal doc on this: https://github.com/kubernetes/cloud-provider-openstack/blob/0dfdb1c5e22f59ee1f09198a0eb68f0716dd1ec5/docs/migrate-to-ccm-with-csimigration.md

**Acceptance criteria**

* Existing Openstack volumes are migrated and can be handled by the CSI driver
* Existing Openstack LoadBalancers are migrated and can be handled by the external CCM

**Comment by moadqassem** [comment]
This ticket is blocked by up-stream

**Comment by xmudrii** [comment]
Duplicate of #5799
/close
